converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java

```diff
@@ -22,4 +26,29 @@ public static MeasureConfig getMeasureConfig(Node node) {
 		String measureId =  node.getValue(MEASURE_ID);
 		return MeasureConfigs.getConfigurationMap().get(measureId);
 	}
+
+	/**
+	 * Determine which measure id-ish value should be used for a given node.
+	 *
+	 * @param node a decoded node
+	 * @return the best available measure id value
+	 */
+	public static String getPrioritizedId(Node node) {
+		MeasureConfig measureConfig = getMeasureConfig(node);
+		return getPrioritizedId(measureConfig);
+	}
+
+	/**
+	 * Find the best available measure id value within the given {@link MeasureConfig}
+	 *
+	 * @param measureConfig a configuration that details a measure
+	 * @return the best available measure id from the given config or a message stating that none were found.
+	 */
+	static String getPrioritizedId(MeasureConfig measureConfig) {
```

Comment:
There already exists `MeasureConfigs#getMeasureId` that does something similar to this.  Think you could combine the approaches even though they seem to prioritize a different ID?

Reply:
Yeah, but it's not a quick fix. There is some usage of static methods in that helper that would make it difficult to test. Refactor gets a little messy given that the helper depends on Node (in converter) and MeasureConfigs static methods (in common) so it may require some shuffling unless I can find a clean way to unknot the dependencies on static methods. I'll ruminate on it for a bit.